testsuites/basic/timesync.py

```diff
@@ -0,0 +1,69 @@
+import re
+
+from assertpy import assert_that
+
+from lisa import Environment, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Windows
+from lisa.testsuite import simple_requirement
+from lisa.tools import Cat, Dmesg
+from lisa.util import get_matched_str
+
+
+@TestSuiteMetadata(
+    area="time",
+    category="functional",
+    description="""
+    This test suite is related with time sync.
+    """,
+    tags=["timesync"],
+)
+class TimeSync(TestSuite):
+    ptp_registered_dmesg_pattern = re.compile(r"PTP clock support registered", re.M)
```

Comment:
It looks they are just string match, no regular expression. Use string const is simpler.